tv.kodi.Kodi.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk9"
+    ],
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=x11",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.Avahi"
+    ],
+    "command": "kodi",
+    "post-install": [
+        "sed -i 's|^KODI_DATA.*|KODI_DATA=${HOME}/.var/app/tv.kodi.Kodi/data|' /app/bin/kodi"
+    ],
+    "modules": [
+        "deps/avahi.json",
+        "deps/bluez.json",
+        "deps/fribidi.json",
+        "deps/fstrcmp.json",
+        "deps/ghostscript.json",
+        "deps/glu.json",
+        "deps/libass.json",
+        "deps/libcdio.json",
+        "deps/lzo2.json",
+        "deps/samba.json",
+        "deps/swig.json",
+        "deps/taglib.json",
+        "deps/tinyxml.json",
+        "deps/udev.json",
+        {
+            "name": "kodi",
+            "buildsystem": "cmake",
```

Comment:
`cmake-ninja`

Reply:
ok

Reply:
For what is ghostscript required; is this dependency really required?